pkg/agent/multicluster/mc_route_controller.go

```diff
@@ -357,7 +582,11 @@ func (c *MCDefaultRouteController) addMCFlowsForSingleCIImp(activeGW *mcv1alpha1
 	}
 	if activeGW.Name == c.nodeConfig.Name {
 		klog.V(2).InfoS("Adding/updating flows to remote Gateway Node for Multi-cluster traffic", "clusterinfoimport", ciImport.Name, "cidrs", allCIDRs)
-		localGatewayIP := net.ParseIP(activeGW.GatewayIP)
+		localGatewayIP := getLocalGatewayIP(activeGW, c.serviceCIDRProvider)
+		if localGatewayIP == nil {
+			klog.V(2).InfoS("Local Gateway IP has not been allocated, skip.", "gateway", activeGW)
```

Comment:
No need to add `.` to the end of a log message.

Reply:
Removed.